packages/mui-material-nextjs/src/v13-pagesRouter/pagesRouterV13Document.tsx

```diff
@@ -37,7 +37,7 @@ export function createGetInitialProps(plugins: Plugin[]) {
 }
 
 export interface DocumentHeadTagsProps {
-  emotionStyleTags: React.JSX.Element[];
+  emotionStyleTags: React.ReactElement[];
```

Comment:
See https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement for why this type didn't make much sense.